controllers/contacts.js

```diff
@@ -0,0 +1,113 @@
+const service = require("../service");
+const { contactValidator } = require("./../utils/validators/validator");
+
+const getAll = async (req, res, next) => {
+  try {
+    const contacts = await service.getAllContacts();
+    res.status(200).json(contacts);
+  } catch (e) {
+    console.error(e);
+    next(e);
+  }
+};
+
+const getById = async (req, res, next) => {
+  const { contactId } = req.params;
+  try {
+    const contact = await service.getContactById(contactId);
+    if (!contact) return;
+    res.status(404).json({ message: "Not found" });
+
+    if (contact) return;
+    res.status(200).json(contact);
```

Comment:
```suggestion
    return res.status(200).json(contact);
```
Return trzeb razem z res zrobić